### PR TITLE
Incomplete missing portfolio deploy

### PR DESCRIPTION
**Pull request by Abebe01**
# Submitting Week 5 Assessment

### ✅ Completed Files

Select the following as confirmation that the files in this repository are complete.

- [x] Career Readiness
- [ x] Coding with AI
- [ x] Interview Questions
- [x ] JavaScript Challenges
- [ x] Ruby Challenges

### ✋ Reminder

Do NOT merge your assessment!

### 📝 Feedback

If you would like specific feedback please leave details here:

I was not able to get my site to deploy. I attempted both methods mentioned via slack but no luck to get the proper node version to run with the app.

### CODE REVIEW

File: coding-with-ai.rb [hunk 1]
```diff
@@ -13,4 +13,4 @@
   result
 end
 
-p totals
\ No newline at end of file
+p totals
```
Comment (elysemontano) [inline coding-with-ai.rb:22]: I am not seeing any content on this section.

**Review by elysemontano — CHANGES_REQUESTED** [review]
Requesting changes:
- Deployed link (happy to help troubleshoot this with you)
- coding with ai section is incomplete

Once you have submitted these changes, please let me know and I will review this assessment.  Thanks!

File: coding-with-ai.rb (hunk 1 above)
Comment (Abebe01) [reply coding-with-ai.rb:22]: Sorry, I don't know how i missed that !

File: coding-with-ai.rb [hunk 2]
```diff
@@ -6,11 +6,17 @@
 
 # Add comments to each line to explain the code.
 
+
+# Prompt - Create a script that processes votes for multiple items. Given a list of votes, make sure to standardize them (convert to lowercase), then compute and record the vote count for each item in a data structure like a dictionary or hash. Finally, display the tally of votes for each item.
```
Comment (elysemontano) [inline coding-with-ai.rb:10]: Awesome work on this prompt!

File: interview-questions.md [hunk 3]
```diff
@@ -8,32 +8,32 @@ Then, research the question to expand on your answer. Even if you feel you have
...
-Your answer:
+Your answer: To implement a TTD workflow means to test everything for a 'good' fail first, then build the code or function to  make the test pass.
 
-Researched answer:
+Researched answer: To implement a TTD workflow means to write a Test (Red phase) to describe the expected behavior of the code. Then write the minimum code to make the test pass (Green phase). Then refactor the code (Refactor phase), to make it as clean and efficent as possible while still passing the test!
 
 3. Why would super be used in a Ruby class?
 
-Your answer:
+Your answer: To call back to a method used in the original parent class (inception).
```
Comment (elysemontano) [inline interview-questions.md:23]: I would recommend expanding this answer in an interview.  Interviewers want to get an idea on the concepts that you know along with how you communicate them, so elaborating whenever possible is extremely valuable.

File: javascript-challenges.test.js [hunk 4]
```diff
@@ -23,21 +35,77 @@ const secretCodeWord3 = "Eccentric"
...
+const arr3 = [5, 5, 5, 5, 4]
 // Expected output: false
-const hand4 = [7, 2, 7, 2, 7]
+const arr4 = [7, 2, 7, 2, 7]
 // Expected output: true
 
 // b) Create the function that makes the test pass.
 
-// Pseudo code:
+const poker = (arr) => {
```
Comment (elysemontano) [inline javascript-challenges.test.js:96]: Nice work, logic looks great!  If you are looking to stretch yourself, consider refactoring the first problem to use a higher order function.

File: ruby-challenges.rb [hunk 5]
```diff
@@ -7,19 +8,79 @@
...
+          end
+        #Create a method to decrease the speed
+          def brake(speed_decrease)
+        #Have the current speed - the input speed to decrease 
+            @current_speed -= speed_decrease
+        #Have the current speed = 0 if it tries to/ goes below 0 to avoid negative integers
+            @current_speed = 0 if @current_speed < 0
+            @current_speed
+          end
+      end
```
Comment (elysemontano) [inline ruby-challenges.rb:67]: Once again, logic looks great!  Just keep an eye on indentation.

**Review by elysemontano — APPROVED** [review]
Awesome work Alex!  I left a few comments to take a look at and consider.  Some things to focus on would be expanding your initial interview answers beyond just one sentence.  Otherwise, this looks great and displays an advanced beginner skill acquisition level.  Keep up the great work!

**Pull request merged**